packages/scripts/utils/license.js

```diff
@@ -339,7 +339,21 @@ function checkDepsInTree( deps, options ) {
 				process.stdout.write( `${ WARNING_TEXT } ${ problem }.\n` );
 			}
 		} else {
-			checkDepLicense( dep.path, licenses );
+			// Ensure the package is actually installed before checking its license.
+			// On Windows, npm ls may return a path for optional deps that aren't installed
+			// due to platform mismatch. If optional and not installed, skip with a warning.
+			const packageJsonPath = dep.path + '/package.json';
+			if (
+				process.platform === 'win32' &&
```

Comment:
Do we need to be specific to the platform, or should this behavior apply broadly?